pkg/minikube/node/node.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/viper"
 
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
this is not right, that error is for kuberentes not for minikube

Reply:
I thought that the error we are looking for is the one that we get in response to the `kubectl delete` Command (`client.CoreV1().Nodes().Delete` call) which is: `Error from server (NotFound): nodes "minikube-m%” not found`.

What would be the correct approach? Do we have another way to check that the node exists?   
Can we assume that if the node doesn’t have an IP address, it doesn’t exist? I tried that in my last commit.